aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -238,6 +263,12 @@ module aptos_framework::aptos_governance {
         // Create and validate proposal metadata.
         let proposal_metadata = create_proposal_metadata(metadata_location, metadata_hash);
 
+        if (is_multi_step_proposal) {
```

Comment:
Why not set this multistep flag inside voting.move instead of here? This could be cleaner since you wouldn't need to duplicate the constants in both modules.

Reply:
I was thinking that if we wanted to set the flag inside `voting.move`, we'd need to add a new create-proposal function to take in an additional `is_multi_step_proposal`. By adding it in the proposal metadata map within the `aptos_governance` module, we implicitly pass in the `is_multi_step_proposal` information without having to add a new function. But we can certainly add a new function in `voting.move` if that's cleaner

Reply:
I think adding a new function in voting module would be cleaner.